ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/AuthoritativeIndexedContentInRemoteTest.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (C) 2013 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.ftest.core.content;
+
+import org.apache.commons.io.IOUtils;
+import org.commonjava.indy.ftest.core.AbstractIndyFunctionalTest;
+import org.commonjava.indy.model.core.RemoteRepository;
+import org.commonjava.indy.test.fixture.core.CoreServerFixture;
+import org.commonjava.test.http.expect.ExpectationServer;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.commonjava.indy.pkg.maven.model.MavenPackageTypeDescriptor.MAVEN_PKG_KEY;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+/**
+ * <b>GIVEN:</b>
+ * <ul>
+ *     <li>Authoritative index of content index enabled</li>
+ *     <li>A remote repo</li>
+ *     <li>Remote path one, will be available before repo set to authoritative index enabled first time</li>
+ *     <li>Remote path two, will be available after repo set to authoritative index enabled first time</li>
+ * </ul>
+ *
+ * <br/>
+ * <b>WHEN:</b>
+ * <ul>
+ *     <li>Fetch first path once but not second one</li>
+ *     <li>Enable authoritative index of the remote repo</li>
+ *     <li>Fetch first and second path</li>
+ *     <li>Disable authoritative index of the remote repo</li>
+ *     <li>Enable authoritative index of the remote repo again</li>
+ * </ul>
+ *
+ * <br/>
+ * <b>THEN:</b>
+ * <ul>
+ *     <li>When repo is authoritative index enabled first time, the first path can be fetched, but the second path can not</li>
+ *     <li>When repo is authoritative index disabled, both path are available</li>
+ *     <li>When repo is authoritative index enabled again, both path are still available</li>
```

Comment:
Again, this makes sense while it's fresh in our minds...can you add an explanation for why it's available the second time, so we can remember later?